tests/phpunit/tests/image/functions.php

```diff
@@ -339,6 +307,24 @@ public function test_load_directory() {
 		}
 	}
 
+	/**
```

Comment:
@hellofromtonya Just a quick question: the `markTestSkipped()` check and call _could_ also be moved to this helper function. Was there any particular reason not to do so ?

Reply:
@jrfnl Whoopsie, thought I had refactored it. Thanks for flagging it. Now resolved in commit https://github.com/WordPress/wordpress-develop/pull/1218/commits/f97d2501f8a9e7b85d48a3b575e55073df5d8587